r124/u_inref/ydb312_gtm8182d.csh

```diff
@@ -0,0 +1,60 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Set white box testing environment to avoid assert failures along with REPLMULTINSTUPDATE error
+#setenv gtm_white_box_test_case_enable 1
+#setenv gtm_white_box_test_case_number 137
+#setenv gtm_repl_instance "mumps.repl"
+
+$MULTISITE_REPLIC_PREPARE 2
+
+setenv gtm_custom_errors /dev/null
+
+echo "# Create the DB"
+$gtm_tst/com/dbcreate.csh mumps 1 -gld_has_db_fullpath >>& dbcreate.out
+if ($status) then
+	echo "DB Create Failed, Output Below"
+	cat dbcreate.out
+	exit -1
+endif
+echo ""
+echo ""
+
+echo "# Start INST1 INST2 replication"
+$MSR START INST1 INST2
+echo ""
+echo "# Stop INST1 INST2 replication"
+$MSR STOP INST1 INST2
+echo ""
+
+setenv path_INST1 `$tst_awk '{-F " "; if ($1" "$2 ~ /INST1 DBDIR/)  print $3}' $tst_working_dir/msr_instance_config.txt`
+
+
+#$GDE SHOW -INSTANCE  >& show.log
+#$GDE CHANGE -INSTANCE -FILE_NAME=\"\" >>& GDEchangeINST1_3.log
+#$MSR SYNC INST1 INST2
+#$MSR RUN INST4 '$MUPIP extract INST4_3_extract.glo'
+$gtm_dist/mumps -run ydb312gtm8182d
```

Comment:
Need a comment in the outref file corresponding to this line indicating what this does.